File: other/fizz_buzz.c
#include <stdio.h>

void Talk(int i)
{
	if (0 == (i % 3) )
		printf("Fizz");

	if (0 == (i % 5))
		printf("Buzz");

	if ((i % 3 ) == 0 || (i % 5 ) == 0)
		printf("\n");
}

int main()
{
	int i;
	for (i = 1 ; i < 100 ; i++)
		Talk(i);
	return 0;
}

File: other/lua/fizz_buzz.lua
require 'io'
function PrintBuzz(i)
	if (i % 3 == 0) then
		io.write("Fizz")
	end
	if (i % 5 == 0) then
		io.write("Buzz")
	end
	if ( ((i % 3) == 0) or ((i % 5) == 0)) then
		io.write("\n")
	end
end

for i=1, 100, 1 do
	PrintBuzz(i)
end
